src/overlays/actors/ovl_Obj_Ocarinalift/z_obj_ocarinalift.h

```diff
@@ -3,17 +3,32 @@
 
 #include "global.h"
 
+#define OBJOCARINALIFT_GET_C(thisx) (((thisx)->params >> 0xC) & 3)
+#define OBJOCARINALIFT_GET_7F(thisx) ((thisx)->params & 0x7F)
+#define OBJOCARINALIFT_GET_1F(thisx) (((thisx)->params >> 7) & 0x1F)
+#define OBJOCARINALIFT_GET_SWITCH_FLAG(thisx) ((thisx)->home.rot.x & 0x7F)
+
 struct ObjOcarinalift;
 
 typedef void (*ObjOcarinaliftActionFunc)(struct ObjOcarinalift*, PlayState*);
 
 typedef struct ObjOcarinalift {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x18];
-    /* 0x015C */ ObjOcarinaliftActionFunc actionFunc;
-    /* 0x0160 */ char unk_160[0x18];
+    /* 0x000 */ DynaPolyActor dyna;
+    /* 0x15C */ ObjOcarinaliftActionFunc actionFunc;
+    /* 0x160 */ f32 unk160;
+    /* 0x164 */ s32 unk164;
+    /* 0x168 */ s32 unk168;
+    /* 0x16C */ s32 unk16C;
+    /* 0x170 */ Vec3s* unk170; // pointer segun mips2c
+    /* 0x174 */ s16 timer;
+    /* 0x176 */ s16 cutsceneTimer;
 } ObjOcarinalift; // size = 0x178
 
+typedef enum ObjOcarinaliftParams {
+    /* 0x0 */ PARAMS_0,
+    /* 0x1 */ PARAMS_1
```

Comment:
Add a prefix to those enum values

Reply:
I think i want other people have input on this, i've done a lot of param enums and i've never been asked to prefix them with the actor overlay name, it seems redundant as the prefix is already present in the Enum's name

Reply:
Yes please add the actors name as a prefix to prevent name collisions. I must not have noticed this in other PRs or I would have brought it up sooner.